mars/worker/utils.py

```diff
@@ -165,3 +168,33 @@ def build_load_key(graph_key, chunk_key):
     if isinstance(chunk_key, tuple):
         chunk_key = '@'.join(chunk_key)
     return '%s_load_memory_%s' % (graph_key, chunk_key)
+
+
+def put_chunk(session_id, chunk_key, data, receiver_ref):
```

Comment:
This function shall be put in ``transfer.py`` and renamed as ``put_remote_chunk`` as it is not referenced by any code in worker module.

Reply:
Fixed.